azure-pipelines.yml

```diff
@@ -57,8 +57,60 @@ jobs:
       make -C docs html
     displayName: 'makedocs'
 
+- job: 'Test_conda'
+  pool:
+    vmImage: 'Ubuntu-16.04'
+  strategy:
+    matrix:
+      Python37:
+        python.version: '3.7'
+    maxParallel: 4
+
+  steps:
+  - bash: echo "##vso[task.prependpath]/usr/share/miniconda/bin"
+    displayName: Add conda to PATH
+
+  - bash: sudo chown -R $USER /usr/share/miniconda
```

Comment:
maybe it doesn't really matter, but this feels very wrong. Instead, conda has a --prefix option so I think you can avoid permissions issues with  
```
conda env create --quiet --prefix ~/condaenv --file environment.yml

source activate ~/condaenv
```

Reply:
this is the recommended way to do it: https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/anaconda?view=azure-devops&tabs=ubuntu-16-04#optional-take-ownership-of-condas-installation-directory

Reply:
only to modify the base environment which you shouldn't be doing since you made a new one. I think the issue is https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/anaconda?view=azure-devops&tabs=ubuntu-16-04#run-pipeline-steps-in-an-anaconda-environment

You can get rid of the chown line, and call ``source activate``in each ``script`` to use the proper environment

Reply:
the conda list does not show the hdf5 lib that was supposed to be installed because of the missing source activate

Reply:
oh :(

Reply:
yep, that fixed it